package.json

```diff
@@ -33,7 +33,9 @@
         "jest": "^25.1.0",
         "jest-circus": "^25.1.0",
         "jest-extended": "^0.11.2",
+        "jest-html-reporter": "^2.8.0",
         "jest-junit": "^10.0.0",
+        "jest-slow-test-reporter": "^1.0.0",
```

Comment:
do we need this?

Reply:
it is used by the commented code that i have.

Reply:
my point that the code is commented, so technically we don't use it, but it's ok

Reply:
we have lots of long running tests now. We can remove it once we fix the issue i created. I can may be added a comment.